### PR TITLE
RIA-3742 notify homeoffice ftpa-appalent-summitted

### PR DESCRIPTION
**Pull request by burale**
### JIRA link (if applicable) ###

[RIA-3742](https://tools.hmcts.net/jira/browse/RIA-3742)

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by divisathyan — DISMISSED** [review]
LGTM

**Review by divisathyan — APPROVED** [review]
LGTM

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -37,6 +37,11 @@ def secrets = [
         secret('home-office-glasgow-email', 'IA_HOME_OFFICE_GLASGOW_EMAIL'),
         secret('home-office-apc-email', 'IA_HOME_OFFICE_APC_EMAIL'),
 
+        secret('ftpa-home-office-bradford-email', 'IA_FTPA_HOME_OFFICE_BRADFORD_EMAIL'),
+        secret('ftpa-home-office-taylor-house-email', 'IA_FTPA_HOME_OFFICE_TAYLOR_HOUSE_EMAIL'),
+        secret('ftpa-home-office-north-shields-email', 'IA_FTPA_HOME_OFFICE_NEWCASTLE_EMAIL'),
```
Comment (divisathyan) [inline Jenkinsfile_CNP:42]: secret variable is north-shields but environment variable is NEWCASTLE.
Would have been better to keep the same name for association

Comment (burale) [reply Jenkinsfile_CNP:42]: This has been the case for all other environment variables, so I kept it the same to avoid confusion .

**Review by mkayad — APPROVED** [review]
LGTM

**Pull request merged**